### PR TITLE
[FIX-21] building docs locally

### LINKED ISSUE #21: repair local doc build
At the moment, it is not possible to build the documentation locally because of several bugs.

### PR DESCRIPTION
**Pull request by szabo137**
Resolves #21.

## Applied changes
* Removed `sorting` argument from `CitationsBibliography`
* Adjusted syntax of the latest version of `makedocs` using the `plugin` keyword
* Fixed typo in docstrings
* Removed docstring of the `QEDbase` module (this feature might be broken)
* Added `QEDbase` as a dependency to docs

### CODE REVIEW

File: src/particle_spinors.jl [hunk 1]
```diff
@@ -3,6 +3,7 @@ import Base.getindex
 
 const SPINOR_VALIDITY_CHECK = Ref(true)
 
+
```
Comment (SimeonEhrig) [inline src/particle_spinors.jl:6]: Please run the formatter script. This removes the extra line.

**Comment by SimeonEhrig** [comment]
Why there are a `make.jl` and `make_local.jl`. Looks like the `make.jl` is not working standalone.

**Comment by SimeonEhrig** [comment]
Please use an "absolute" path here:
https://github.com/QEDjl-project/QEDbase.jl/blob/b3a7ad63d9e95e1148be873c354ac46692bbed77/docs/make_local.jl#L3

```julia
project_path = Base.Filesystem.joinpath(Base.Filesystem.dirname(Base.source_path()), "..")
```

File: src/particle_spinors.jl (hunk 1 above)
Comment (szabo137) [reply src/particle_spinors.jl]: Done in 467cfee.

**Comment by szabo137** [comment]
> Please use an "absolute" path here:
> 
> https://github.com/QEDjl-project/QEDbase.jl/blob/b3a7ad63d9e95e1148be873c354ac46692bbed77/docs/make_local.jl#L3
> 
> ```julia
> project_path = Base.Filesystem.joinpath(Base.Filesystem.dirname(Base.source_path()), "..")
> ```

Oh, this is some ancient leftover. 0945480 removes that.

**Comment by SimeonEhrig** [comment]
Looks good. The documentation builds and didn't saw anything strange in the rendered documentation.

**Review by SimeonEhrig — APPROVED** [review]

**Pull request merged**